VirtoCommerce.Storefront.IntegrationTests/Infrastructure/MoneyJsonConverter.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using VirtoCommerce.Storefront.Model;
+using VirtoCommerce.Storefront.Model.Common;
+
+namespace VirtoCommerce.Storefront.IntegrationTests.Infrastructure
+{
+    public class MoneyJsonConverter : JsonConverter<Money>
+    {
+        public override bool CanWrite => true;
+        public override bool CanRead => true;
+
+        public override Money ReadJson(JsonReader reader, Type objectType, Money existingValue, bool hasExistingValue, JsonSerializer serializer)
+        {
+            var obj = JObject.Load(reader);
+
+            var currencyCode = obj["currency"]["code"].Value<string>();
+            var amount = obj["amount"].Value<decimal>();
+
+            // TODO: here we should try to find incoming currency code inside available currencies,
```

Comment:
What you think about removing this comments?

Reply:
removed